gateway/main.ts

```diff
@@ -42,32 +35,35 @@ class App {
           }
         }
       } catch (e) {
-        this.logger.error(`Exception:`, e);
+        this.logger.error(`Exception: ${e}`);
```

Comment:
didn't we change in the API from `${e}` to `, e` to improve readability?

Reply:
Yes, but it was really spamming the log with huge HTTP exceptions 🤣

Reply:
I could make sense to write the log to a file ... I'll do this.